packages/commercetools/theme/components/checkout/CartPreview.vue

```diff
@@ -122,25 +130,31 @@ export default {
     SfCharacteristic,
     SfInput
   },
-  props: {
-    order: {
-      type: Object,
-      default: () => ({})
-    },
-    shippingMethods: {
-      type: Array,
-      default: () => []
-    },
-    paymentMethods: {
-      type: Array,
-      default: () => []
-    }
-  },
-  data() {
+  setup(props) {
+    const { chosenShippingMethod } = useCheckout()
+    const { cart, removeFromCart, updateQuantity } = useCart()
+    const listIsHidden = ref(false)
+    const promoCode = ref('')
+    const showPromoCode = ref(false)
+    const products = computed(() => getCartProducts(cart.value, ['color', 'size']))
+    const totalItems = computed(() => getCartTotalItems(cart.value))
+    const subtotal = computed(() => getCartSubtotalPrice(cart.value))
+    const total = computed(() => getCartTotalPrice(cart.value))
+
     return {
-      promoCode: "",
-      showPromoCode: false,
-      listIsHidden: false,
+      totalItems,
+      listIsHidden,
```

Comment:
I like the convention of keeping all local  ui-related switches in `ui` object. WDYT about that?